Makefile

```diff
@@ -89,3 +91,14 @@ uninstall: scripts_uninstall modules_uninstall
 lunatik_sym.h: $(LUA_API)
 	${shell ./gensymbols.sh $(LUA_API) > lunatik_sym.h}
 
+moontastik_install_%:
+	[ $* ] || (echo "usage: make moontastik_install_TARGET" ; exit 1)
+	wget https://github.com/luainkernel/moontastik/releases/download/${MOONTASTIK_RELEASE}/moontastik_lua.zip -O moontastik_lua.zip
+	[ -d moontastik_lua ] && ${RM} -r moontastik_lua || true
+	unzip moontastik_lua.zip
+	cd moontastik_lua/"$*" && ./install.sh ; cd -
+
+moontastik_uninstall_%:
+	[ $* ] || (echo "Which target would you like to uninstall?" && exit 1)
```

Comment:
Can you also change this to "usage ..."?

Reply:
Sorry, didn’t think about this one…